components/io_expander/esp_io_expander_tca95xx_16bit/README.md

```diff
@@ -0,0 +1,40 @@
+# ESP IO Expander Chip TCA9539 and TCA9555
+
```

Comment:
Could you add this line?
`[![Component Registry](https://components.espressif.com/components/espressif/esp_io_expander_tca95xx_16bit/badge.svg)](https://components.espressif.com/components/espressif/esp_io_expander_tca95xx_16bit)`

It will add a nice badge with latest version in IDF Compnent Registry